game.js

```diff
@@ -122,22 +122,34 @@ class Game {
                 && clock.pause && clock.pause.paused && clock.pause.pause_control
                 && !clock.pause.pause_control["stone-removal"] && !clock.pause.pause_control.system && !clock.pause.pause_control.weekend
                 && !clock.pause.pause_control["vacation-" + clock.black_player_id] && !clock.pause.pause_control["vacation-" + clock.white_player_id]) {
-                if (config.DEBUG) this.log("Pausing not allowed. Resuming game.");
-                this.resumeGame();
+                if (config.DEBUG) {
+                    const noPauseMg = "Pausing not allowed. Resuming game.";
+                    this.log(noPauseMg);
+                    this.sendChat(noPauseMg)
+                    this.resumeGame();
+                }
             }
 
             if (config.nopauseranked && this.state.ranked && clock.pause && clock.pause.paused && clock.pause.pause_control
                 && !clock.pause.pause_control["stone-removal"] && !clock.pause.pause_control.system && !clock.pause.pause_control.weekend
                 && !clock.pause.pause_control["vacation-" + clock.black_player_id] && !clock.pause.pause_control["vacation-" + clock.white_player_id]) {
-                if (config.DEBUG) this.log("Pausing not allowed for ranked games. Resuming game.");
-                this.resumeGame();
+                if (config.DEBUG) {
+                    const noPauseMg = "Pausing not allowed for ranked games. Resuming game.";
+                    this.log(noPauseMg);
+                    this.sendChat(noPauseMg)
```

Comment:
Same as previous msg. Also duplicate code here so this could be a function.

Reply:
same as https://github.com/online-go/gtp2ogs/pull/222#discussion_r402074409